src/hatch_openzim/utils.py

```diff
@@ -0,0 +1,64 @@
+import re
+import subprocess
+from functools import lru_cache
+from typing import List
+
+from packaging.specifiers import SpecifierSet
+from packaging.version import Version
+
+REMOTE_REGEXP = re.compile(
+    r"""^(?:git@|https:\/\/)github.com[:\/](.*?)\/(.*?)(?:.git)?$"""
+)
+
+
+@lru_cache(maxsize=None)
+def get_github_project_homepage(root: str, remote: str = "origin") -> str:
+    git_remote_url = (
+        subprocess.check_output(
```

Comment:
Although `git` is most likely installed on the computer that will run the hatch build, this is not a requirement and **might** be an issue if the user downloaded the source (from github) or doesn't have `git` installed system wide (windows).
I'm not comfortable with to external dependencies (as we have in scraperlib) so given this information is avaible in the INI git config, I think it would be best to read it direcly from there

Reply:
and 3.8 already has `subprocess.run()` with `check` and `text` so there's no excuse for `check_output` nor `decode()`

Reply:
Works for me 👍